package.json

```diff
@@ -24,10 +24,11 @@
     "hexo-cli": "^1.0.1"
   },
   "dependencies": {
-    "hexo": "^3.2.0",
+    "hexo": "^3.2.2",
     "hexo-cli-extras": "^1.0.3",
     "hexo-deployer-git": "^0.1.0",
     "hexo-filter-pathfix": "^0.1.2",
+    "hexo-generator-alias": "hexojs/hexo-generator-alias",
```

Comment:
We need to install this from github because there isn't an npm release that supports hexo 3.x yet.